easybuild/easyconfigs/v/Valgrind/Valgrind-3.16.1-gompi-2019b.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = 'ConfigureMake'
+
+name = 'Valgrind'
+version = '3.16.1'
+
+homepage = 'https://valgrind.org'
+description = "Valgrind: Debugging and profiling tools"
+
+toolchain = {'name': 'gompi', 'version': '2019b'}
+toolchainopts = {'optarch': False}
```

Comment:
@tstrempel why setting optarch to False? I tested it without this option and it builds fine.

Reply:
@smoors I took this option from an older EasyConfig. As it builds without it, I'll remove it.